config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -434,11 +434,11 @@ presubmits:
         - --timeout=420m
         - --stage=gs://kubernetes-security-prow/ci/pull-security-kubernetes-e2e-aks-engine-azure
-        image: gcr.io/k8s-testimages/kubekins-e2e:v20190219-caa48e6c3-master
+        image: ritazh/kubekins-e2e:v20190220-d69e2d7-dirty-master
```

Comment:
should we use some tag build from `gcr.io/k8s-testimages`?

Reply:
Sorry forgot to change this back. Just restored to the old image, but will need to create another PR once we get a new kubekins-e2e image after this PR is merged.